opentimelineio/adapters/cmx_3600.py

```diff
@@ -104,8 +104,15 @@ def add_clip(self, line, comments, rate=24):
         comment_handler = CommentHandler(comments)
         clip_handler = ClipHandler(line, comment_handler.handled, rate=rate)
         clip = clip_handler.clip
+        reel = clip_handler.reel
+        if reel != 'AX':
+            if clip.metadata.get('cmx_3600', None) is None:
```

Comment:
`dict.get` will default to `None`, so this can be rewritten:
```suggestion
            if clip.metadata.get('cmx_3600') is None:
```
`None` is also falsy so you _could_ write it:
```suggestion
            if not clip.metadata.get('cmx_3600'):
```
...however it looks like there's already `is None` precedent in the file so  ¯\\\_(ツ)\_/¯.